packages/parse5/lib/tokenizer/index.test.ts

```diff
@@ -1,24 +1,18 @@
 import * as parse5 from 'parse5';
 import { Tokenizer } from 'parse5/dist/tokenizer/index.js';
-import type { Token } from 'parse5/dist/common/token';
 import { generateTokenizationTests } from 'parse5-test-utils/utils/generate-tokenization-tests.js';
 
 const dataPath = new URL('../../../../test/data/html5lib-tests/tokenizer', import.meta.url);
+const tokenizerOpts = {
+    sourceCodeLocationInfo: true,
+};
 
-generateTokenizationTests('tokenizer', 'Tokenizer', dataPath.pathname, ({ errors }) => {
-    const tokenizer = new Tokenizer({
-        sourceCodeLocationInfo: true,
-        onParseError(err): void {
```

Comment:
The `onParseError` method for the tokenizer moved to the handler. This is not a breaking change for anyone not explicitly using the tokenizer (and there are a lot of breaking changes here for those that are).